README.md

```diff
@@ -39,7 +39,6 @@ You can also run Minerva using docker. To run Minerva in docker, follow the inst
 ```sh
 docker compose up
 ```
-
```

Comment:
Please bring back this empty line :)

Also, let's avoid making irrelevant changes to pull requests.